stats/view/view.go

```diff
@@ -90,6 +93,12 @@ func (v *View) canonicalize() error {
 	sort.Slice(v.TagKeys, func(i, j int) bool {
 		return v.TagKeys[i].Name() < v.TagKeys[j].Name()
 	})
+	sort.Float64s(v.Aggregation.Buckets)
+	for _, b := range v.Aggregation.Buckets {
+		if b < 0 {
+			return ErrNegativeBucketBounds
```

Comment:
We just want to log a warning in case of negative bounds and silently remove 0 bounds, counterpart: https://github.com/census-instrumentation/opencensus-java/blob/master/api/src/main/java/io/opencensus/stats/BucketBoundaries.java#L81-L90.

Reply:
I think it is good to return an error if the API already had the error.